docs/ios/open-source/automatic-instrumentation/network-capture.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Network Capture
+description: Automatically monitor network requests and responses in your iOS app
+sidebar_position: 4
+---
+
+# Network Capture
+
+The Embrace SDK's `NetworkCaptureService` automatically captures HTTP/HTTPS network requests and responses in your app, providing visibility into API performance, error rates, and data transfer volumes.
+
+## How Network Capture Works
+
+The network capture service instruments URLSession-based networking in your app by swizzling key methods in the URLSession and URLSessionTask classes. This allows Embrace to create OpenTelemetry spans for each network request that capture:
+
+- Request URL, method, and headers
+- Response status code and headers
+- Request and response body size
+- Network timing metrics (DNS lookup, TLS handshake, time to first byte, etc.)
+- Error information (if applicable)
+
+This data helps identify slow APIs, track error rates, and understand the network performance impact on your app.
```

Comment:
can we add an :::info::: note here about how the SDK doesn't scrub PII by default, so devs should configure the below to exclude it? I know it's noted below but good to make it known conceptually at top as well